openapi-config.json

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
Do we plan to commit this file with the repo? It's only used to generate the client and we don't need to retain it afterwards. Or can we move it to a different directory to avoid cluttering the top-level namespace?

Reply:
Deleted it